include/cpuinfo.h

```diff
@@ -353,6 +353,19 @@ enum cpuinfo_uarch {
 	cpuinfo_uarch_palm_cove = 0x0010020B,
 	/** Intel Sunny Cove microarchitecture (10 nm, Ice Lake). */
 	cpuinfo_uarch_sunny_cove = 0x0010020C,
+	/** Intel Cypress Cove microarchitecture (14 nm, Rocket Lake) */
+	cpuinfo_uarch_cypress_cove = 0x0010020E,
+	/** Intel Golden Cove microarchitecture (Intel 7, Alder Lake P-Cores) */
+	cpuinfo_uarch_golden_cove = 0x0010020F,
+	/** Intel Gracemont Cove microarchitecture (Intel 7, Alder/Raptor Lake E-Cores) */
```

Comment:
E-Core names end in 'mont'.  Remove 'cove' which is the naming for P-Cores.

Reply:
Thanks @fbarchard, I removed 'cove' from the end of Gracemont and Crestmont.